src/Symfony/Component/Notifier/Transport/NullTransport.php

```diff
@@ -20,8 +23,18 @@
  */
 class NullTransport implements TransportInterface
 {
+    private $dispatcher;
+
+    public function __construct(EventDispatcherInterface $dispatcher = null)
+    {
+        $this->dispatcher = LegacyEventDispatcherProxy::decorate($dispatcher);
```

Comment:
`$this->dispatcher = class_exists(\Symfony\Component\EventDispatcher\Event::class) ? LegacyEventDispatcherProxy::decorate($dispatcher) : $dispatcher;`